jvm-packages/xgboost4j-spark/src/test/scala/ml/dmlc/xgboost4j/scala/spark/XGBoostClassifierSuite.scala

```diff
@@ -35,7 +35,8 @@ class XGBoostClassifierSuite extends FunSuite with PerTest {
       "eta" -> "1",
       "max_depth" -> "6",
       "silent" -> "1",
-      "objective" -> "binary:logistic")
+      "objective" -> "binary:logistic",
+      "batchSize" -> "200")
```

Comment:
In the key-value way to set params, it should be ```batch_size```.

Reply:
done